spec.html

```diff
@@ -2981,7 +2981,7 @@
 
     <!-- es6num="7.1.1" -->
     <emu-clause id="sec-toprimitive" aoid="ToPrimitive">
-      <h1>ToPrimitive ( _input_ [, _PreferredType_] )</h1>
+      <h1>ToPrimitive ( _input_ [ , _PreferredType_] )</h1>
```

Comment:
There doesn't seem to be consistent spacing inside brackets in the spec. I made minimal changes to create consistency with a space between `[` and `,`.

I'll make this change here, but I'm not confident I can make the entire spec consistent in this regard.